pwa/src/templates/components/ComponentsTemplate.module.css

```diff
@@ -18,8 +20,11 @@
   margin-inline-end: var(--nlportal-space-block-xl);
 }
 
-.componentsGrid {
-  display: grid;
-  grid-gap: var(--nlportal-space-block-md);
-  grid-template-columns: 1fr 1fr;
+.buttongroup {
+  display: flex;
+  align-items: center;
+}
+
+.buttongroup > button:not(:last-child) {
```

Comment:
Might not be possible with the way we import the buttons, but generally speaking we'd like to avoid calling elements directly (so instead of `button` we would want to call `.buttonClassName`).